.gitignore

```diff
@@ -4,7 +4,6 @@ _build/
 dev.db
 site_media/
 static/dist/
-ti.xml
```

Comment:
I don't think we want to commit the `ti.xml` file, do we?

Reply:
I was downloading it each time in the entrypoint script for the dev dockerfile, which slowed down the build a bit. 

I kind of assumed that since the file is for dev only that we don't really need the "freshest" data. Is that the concern?

I can add a step to remove it from the production dockerfile...